Explorer/Assets/DCL/PerformanceAndDiagnostics/Optimization/Pools/ComponentPool.cs

```diff
@@ -4,39 +4,53 @@
 
 namespace DCL.Optimization.Pools
 {
-    public class ComponentPool<T> : IComponentPool<T> where T: class, new()
+    /// <summary>
+    /// Entry point for Thread-safe pools dedicated to SDK/ECS components
+    /// </summary>
+    public static class ComponentPool
```

Comment:
Maybe replace with Singleton if you decided to move it as static? Singleton can be mocked with an Interface, static not

Reply:
It's just a container for other classes, it has no methods 😄